scripts/cord19/neo4j_loader.py

```diff
@@ -0,0 +1,43 @@
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
```

Comment:
rename into a verb phrase, e.g., `load_citation_graph_into_neo4j` for clarity.

I like scripts that have `main` to be verb phrases.